pkg/sfu/prober.go

```diff
@@ -169,6 +169,7 @@ func (p *Prober) AddCluster(desiredRateBps int, expectedRateBps int, minDuration
 		return
 	}
 
+	fmt.Printf("adding cluster, desired: %d, expected: %d, min: %d, max: %d\n", desiredRateBps, expectedRateBps, minDuration.Milliseconds(), maxDuration.Milliseconds()) // REMOVE
```

Comment:
would be great to use logger here.

Reply:
Disobeyed my own rules. That's why I put remove marker and forgot to remove it 🙂. There is already a logger there. Will remove this.